docs/book/appendix-default-aws-settings.md

```diff
@@ -17,8 +17,9 @@ gives this the skill requests's `context.System.device.deviceId` field, by defau
 
 ## Lambda
 
-* Lambda runtime should use `nodejs10.x` or later.
-  * We can check the runtime EOL in [the AWS document](https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html).
+* Runtime is set to `nodejs12.x` (minimum version is `nodejs10.x`).
```

Comment:
Maybe add by default: "set to `nodejs12.x` by default ..."

Reply:
This file is dedicated to default settings, so we are fine :)